apps/web/components/ui/LinkText.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+
+interface ILinkTextProps {
+  href: string;
+  children: React.ReactNode;
+  classNameChildren?: string;
+}
+// This component had to be made in order to make i18n work with next/link
+// Reference: https://github.com/i18next/react-i18next/issues/1090#issuecomment-615426145
+
```

Comment:
```suggestion
/**
 * This component had to be made in order to make i18n work with next/link
 * @see https://github.com/i18next/react-i18next/issues/1090#issuecomment-615426145
 **/
```

Modded comment so it actually displays on hover:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/3504472/186539039-7cb7d3f6-042f-4bb8-a3c6-0a1c1006e976.png">

Reply:
Awesome